WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/SupportForm/SupportForm.swift

```diff
@@ -20,8 +20,8 @@ final class SupportFormHostingController: UIHostingController<SupportForm> {
         hidesBottomBarWhenPushed = true
     }
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
+    override func viewWillAppear(_ animated: Bool) {
```

Comment:
Previously we were creating the Zendesk identity only once through the app lifecycle, is there any concern regarding recreating it each time the view appears? When thinking about switching sites, or contacting support after an app crash I can't see any problem with it, we'll just return early if there's one or recreate it, but I though was worth raising it 🤔

Reply:
Given that it's already got a check for whether one already exists, I don't see any concern with doing it repeatedly, but yes I had the same thought and checked that too 👍